include/deal.II/lac/constraint_matrix.h

```diff
@@ -1089,6 +1089,18 @@ class ConstraintMatrix : public Subscriptor
    *
    * @ingroup Exceptions
    */
+  DeclException1 (ExcColumnNotStoredHere,
+                  size_type,
+                  << "The index set given to this constraint matrix indicates "
+                  << "constraints for degree of freedom " << arg1
+                  << " should not be stored by this object, but a constraint "
+                  << "uses this degree of freedom.");
```

Comment:
I don't think I understand this completely. Are you saying that a constraint *references* a DoF that is not part of the index set? (E.g., a constraint for a locally owned DoF references a locally *relevant* DoF, but the index set only includes the locally *active* DoFs?)

If so, could you rephrase the error message and maybe also include the index of the *constrained* DoF where you encounter the invalid index?